app/controllers/well_known/keybase_proof_config_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module WellKnown
+  class KeybaseProofConfigController < ActionController::Base
+    def show
+      render json: {}, serializer: ProofProvider::Keybase::ConfigSerializer
```

Comment:
This seems strange?

Reply:
The serializer does not use a specific object, but uses various globals. See also: REST::InstanceSerializer